src/test/resources/config/application.yml

```diff
@@ -60,9 +71,7 @@ spring:
     h2:
         console:
             enabled: false
-server:
```

Comment:
why this section was removed?

Reply:
ServerProperties is used instead of JHipsterProperties